packages/ability-attributes/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@stardust-ui/ability-attributes",
+  "description": "Accessibility attributes schema for Stardust UI",
+  "version": "0.38.0",
+  "author": "Marat Abdullin <marata@microsoft.com>",
+  "bugs": "https://github.com/stardust-ui/react/issues",
+  "dependencies": {
+    "ability-attributes": "^0.0.4"
+  },
+  "devDependencies": {
+    "ability-attributes-generator": "^0.0.4"
+  },
+  "files": [
+    "dist"
+  ],
+  "homepage": "https://github.com/stardust-ui/react/tree/master/packages/ability-attributes",
+  "jsnext:main": "dist/es/index.js",
+  "license": "MIT",
+  "main": "dist/commonjs/index.js",
+  "module": "dist/es/index.js",
+  "peerDependencies": {
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "stardust-ui/react.git",
+  "scripts": {
+    "build": "npm run schema && gulp bundle:package:no-umd --package ability-attributes",
+    "schema": "allyschema -c \"process.env.NODE_ENV !== 'production'\" schema.json > ./src/schema.ts",
+    "preinstall": "npm run build",
+    "prepare": "npm run build"
```

Comment:
If we will add watch mode `preinstall` and `prepare` are not required anymore I think